### PR TITLE
Boilerplate Content Pack

### PR DESCRIPTION
**Pull request by fvigo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Boilerplate Pack, Integration and Script for development purposes

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — COMMENTED** [review]
I think we can remove all of this .gitignore files

File: Packs/Boilerplate/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Boilerplate",
+    "description": "Boilerplate for building new content",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "devrel@paloaltonetworks.com",
```
[the remark this answers is not in the document]
Comment (fvigo) [reply Packs/Boilerplate/pack_metadata.json:8]: Good point, I put some dummy values for now, if they don't replace them we should catch those during review.

**Review by ShahafBenYakir — COMMENTED** [review]
Can we choose another name than a boilerplate? Maybe `getting started`? or `starter pack`? or something of that sort?

**Comment by roysagi** [comment]
> Can we choose another name than a boilerplate? Maybe `getting started`? or `starter pack`? or something of that sort?

I agree @fvigo - maybe something like `BaseIntegration` and `BaseScript`?

**Comment by fvigo** [comment]
OK, let's agree on the names first.

Pack Name: `StarterPack`
Integration Name: `BaseIntegration`
Script Name: `BaseScript`

Please agree or disagree :)

CC @ShahafBenYakir @roysagi @anara123

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**